### PR TITLE
spelling mistake

### PR DESCRIPTION
**Pull request by orena1**
spelling mistake

### CODE REVIEW

File: src/bokeh/models/tools.py [hunk 1]
```diff
@@ -1606,7 +1606,7 @@ class PolyDrawTool(PolyTool, Drag, Tap):
       to add each subsequent vertex, to finalize the draw action double tap to
       insert the final vertex or press the <<esc> key.
```
Comment (mosc9575) [inline src/bokeh/models/tools.py]:
```suggestion
      insert the final vertex or press the <<esc>> key.
```

Here is an inconsistency, too.

**Comment by bryevdv** [comment]
So I actually decided to check, and all the other special key instances are done differently:
```
docs/bokeh/source/docs/user_guide/interaction/tools.rst
272:    To make multiple selections, press the SHIFT key. To clear the
273:    selection, press the ESC key.
304:    To make a multiple selection, press the SHIFT key. To clear the selection,
305:    press the ESC key.
350:    multiple selection, press the SHIFT key. To clear the selection, press the
372:    To make a multiple selection, press the SHIFT key. To clear the selection,
373:    press the ESC key.
842:  Tap a box to select it, then press the BACKSPACE key while the mouse is
904:  Tap a line or patch to select it then press the BACKSPACE key while the
1020:  the final vertex or press the ESC key.
1027:  Tap a patch/multi-line to select it, then press the BACKSPACE key while
1067:  and add a point, double tap. Otherwise press the ESC key to cancel.
```

There are actually two instances in `tool.py` with the angle brackets:
```
src/bokeh/models/tools.py
1607:      insert the final vertex or press the <<esc> key.
1710:      and add a point double tap otherwise press the <<esc> key to cancel.
```

@orena1 Would you actually mind updating both the instances in this file to look like the others (i.e. to be written simlpy `ESC` rather than `<<esc>>`)

**Comment by orena1** [comment]
Hi @bryevdv, yeah sure, I've replaced all <<x>> keys with X

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @orena1 !

**Pull request merged**